pkg/volume/empty_dir/empty_dir.go

```diff
@@ -321,15 +321,7 @@ func (ed *emptyDir) TearDownAt(dir string) error {
 }
 
 func (ed *emptyDir) teardownDefault(dir string) error {
-	tmpDir, err := volume.RenameDirectory(dir, ed.volName+".deleting~")
-	if err != nil {
-		return err
-	}
-	err = os.RemoveAll(tmpDir)
-	if err != nil {
-		return err
-	}
-	return nil
+	return volume.RenameWithFallback(dir, ed.volName+".deleting~")
```

Comment:
This PR is a semantic change - it used to rename and them remove.  Now it is just rename.